src/components/hotelShow.js

```diff
@@ -6,7 +6,7 @@ import RoomCard from './roomCard';
 import Loader from './Loader';
 import '../assets/styles/hotelShow.css';
 
-export default function HotelShow() {
+export default HotelShow() = () =>{
```

Comment:
- Wrong syntax. Can you change this to const HotelShow = () => {}
Then export default HotelShow